volumes/client/scenarios.yml

```diff
@@ -182,3 +182,33 @@ scenarios:
         response:
           status_code: 403 # Forbidden
           content: 700
+
+  - name: success_tablet_android_return_only_tiles_that_pass_filter
+    # This is not a working test but only for documentation purpose
+    description: Test that Contile only returns tiles that pass the filter.
+    steps:
+      - request:
+          method: GET
+          path: '/v1/tiles'
+          headers:
+            # Contile maps the User-Agent Header value to os-family and form-factor parameters
+            # The following value will result in os-family: android and form-factor: phone(android-tablets falls under 'phones')
+            # This won't return a 200 response but a 500 as contile has cached the response from the previous scenario with
+            # os-family: android and form-factor: phone
+            - name: User-Agent
+              value: 'Mozilla/5.0 (Android; Tablet; rv:92.0) Gecko/92.0 Firefox/92.0'
+            # The response below returned only 1 tile because the other tile (.NET) didn't pass the filter specification in the adm_settings.json
+        response:
+          status_code: 200
```

Comment:
Please move the above comment about the `status_code` code here and refer to the name of the scenario, so it's easier to find: `error_phone_android_reqwest_error`.